tests/test_conda_subprocess.py

```
@@ -43,9 +48,12 @@
         )
 
     def test_check_output_env_name(self):
+        output_new = check_output("which python", prefix_name=self.env_name, universal_newlines=True)
```

Ruff (0.8.2)

Line too long (101 > 88)

(E501)